src/components/gameshow/RightBar.jsx

```diff
@@ -0,0 +1,248 @@
+RightBar.jsx;
+import { fetchGameDetails, fetchGameElements } from '../../api/api-fetch';
+import { dlcURL } from '../../api/api-url';
+import { useEffect, useState } from 'react';
+import ageRating from './AgeRatingLogo';
+import { Tooltip } from 'flowbite';
+import storeLogo from './StoreLogo';
+import platformLogo from './PlatformLogo';
+import DlcComponent from './DlcComponent';
+import perc2color from './MetascoreColor';
+
+export default function RightBar({ gameId }) {
+  const [game, setGame] = useState([]);
+  const [isLoaded, setIsLoaded] = useState(false);
+  const [dlc, setDlc] = useState([]);
+  const [retro, setRetro] = useState([]);
+  useEffect(() => {
+    const controller = new AbortController();
+    const signal = controller.signal;
+    fetchGameDetails({
+      gameId,
+      setter: setGame,
+      setLoaded: setIsLoaded,
+      signal,
+    });
+    fetchGameElements({
+      parameter: dlcURL,
+      gameId,
+      setter: setDlc,
+      setLoaded: setIsLoaded,
+      signal,
+    });
+    return () => controller.abort();
+  }, [gameId]);
+
+  // set the tooltip content element
+  const targetEl = document.getElementById('tooltipContent');
+  // set the element that trigger the tooltip using hover or click
+  const triggerEl = document.getElementById('tooltipButton');
+  const options = {
+    placement: 'top',
+    triggerType: 'hover',
+  };
+  const tooltip = new Tooltip(targetEl, triggerEl, options);
```

Comment:
Pas besoin de stocker dans une `const` si tu l'utilise pas ensuite
```suggestion
  new Tooltip(targetEl, triggerEl, options);
```
suffit